recipe/meta.yaml

```
@@ -1,42 +1,60 @@
 {% set name = "inequality" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e3ad59ece8d25ecfc7c7e94efe59ead0e4ffb76ff72befd18a60e10caba6fd54
+  sha256: 2010253c26cfe3745d36beb02aeff5503e1b21cb06e1d52aa7f13c37acd9adf1
 
 build:
-  noarch: python
+  skip: true  # [py<311]
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools >=61.0
+    - setuptools_scm >=6.2
+    - toml
+    - wheel
   run:
-    - python >=3.5
-    - scipy >=0.11
-    - numpy >=1.3
+    - python
+    - libpysal >=4.12
+    - matplotlib >=3.8
+    - numpy >=1.26
+    - pandas >=2.1
+    - scipy >=1.12
 
 test:
+  source_files:
+    - inequality/tests
   imports:
     - inequality
+  requires:
+    - pip
+    - pytest
+    - seaborn
+  commands:
+    - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - pytest -vv inequality/tests
 
 about:
   home: https://github.com/pysal/inequality
```

```suggestion
  home: https://pysal.org/inequality
```
This URL is set as Home in the GitHub project, but in _pyproject_ is listed the previous one. I'm not really sure which to use?

---

need to use this one: https://github.com/pysal/inequality/blob/v1.1.2/pyproject.toml#L37